### PR TITLE
Updates the implicit code sample

### PR DESCRIPTION
**Pull request by bretterer**
Updates the quickstart guide for php generic implicit

### CODE REVIEW

File: _source/quickstart/php/generic-implicit.md [hunk 1]
```diff
@@ -3,83 +3,219 @@ layout: quickstart_partial
...
+Now that we have ignored any requests for `OPTIONS` (prefetch requests) and made sure that the authorization header 
+is present, we can not begin our verification process.  First, we set up some variables to store some information in.
+ Next we extract the authentication type and the token from the Authorization header then we need to make sure that 
+ the authentication type is a Bearer token.
+ 
 ```php
 <?php
 
-require __DIR__ . '/vendor/autoload.php';  // This should be adjusted to be the autoload file from your vendor folder.
+// PREVIOUS CODE HERE, EXCLUDED FOR SPACE
```
Comment (jmelberg-okta) [inline _source/quickstart/php/generic-implicit.md]: Suggestion: I find that `...` implies that there is usually code we are omitting for space, and is a little cleaner than adding a comment.

File: _source/quickstart/php/generic-implicit.md [hunk 2]
```diff
@@ -3,83 +3,219 @@ layout: quickstart_partial
...
+
+$authType = null;
+$authToken = null;
 
-$jwt = 'eyJhbGciOiJSUzI1Nqd0FfRzh6X0ZsOGlJRnNoUlRuQUkweVUifQ.eyJ2ZXIiOjEsiOiJwaHBAb2t0YS5jb20ifQ.ZGrn4fvIoCq0QdSyA';
+// Extract the auth type and the data from the Authorization header.
+list( $authType, $authToken ) = explode( " ", $_SERVER['HTTP_AUTHORIZATION'], 2 );
+
+// If the Authorization Header is not a bearer type, return a 401.
+if ($authType != 'Bearer' ) {
```
Comment (jmelberg-okta) [inline _source/quickstart/php/generic-implicit.md]: nit: Spacing -> `( $authType != 'Bearer' )`

File: _source/quickstart/php/generic-implicit.md [hunk 3]
```diff
@@ -3,83 +3,219 @@ layout: quickstart_partial
...
-### Extra Notes
-The library currently validates the JWT signature based on the issuer and discovery method, based on the keys from the meta-data URI. An exception will be thrown if the JWT contains an invalid signature, is expired, or was issued in the future. You will need to validate manually the nonce and audience on your own after receiving the claims. An example of this is provided below:
+If the verify method was successful, we will now have access to all the claims of the token. We need to check and 
+make sure that the `cid` (ClientId) from the token matches what our ClientId is from the authoriztation server.
 
 ```php
 <?php
 
-$claims = $jwt->getClaims();
+// PREVIOUS CODE HERE, EXCLUDED FOR SPACE
```
Comment (jmelberg-okta) [inline _source/quickstart/php/generic-implicit.md]: It was clear to me that we are adding this code as we go, so we can probably remove these comments.

File: _source/quickstart/php/generic-implicit.md (hunk 1 above)
Comment (bretterer) [reply _source/quickstart/php/generic-implicit.md]: Was trying to keep it as valid php to prevent copy/paste gremlins

**Pull request merged**